packages/loader/src/custom-file-loaders/Texture2D.ts

```diff
@@ -0,0 +1,41 @@
+import { Engine, Texture2D } from "@oasis-engine/core";
+import { loadImageBuffer } from "../Util";
+import { BufferReader } from "./utils/BufferReader";
+
+export class Texture2DDecoder {
```

Comment:
This file name is `Texture2D` , duplicate with `engine/core`'s `Texture2D`, this is not good! 
You can named this file with `Texture2DDecoder `